code/tracing/categories.h

```diff
@@ -0,0 +1,92 @@
+

```

Comment:
Yes, that's how it's supposed to work. That should allow to declare the tracepoint ahead of time without using strings for identifying them which should help if another tracing framework needs that.